python/doc/usecases/use_case_cantilever_beam.rst

```diff
@@ -236,27 +236,6 @@ Examples based on this use case
    /auto_meta_modeling/kriging_metamodel/plot_kriging_beam_trend
 
 
-.. raw:: html                                                             
-
-    <div class="sphx-glr-thumbcontainer" tooltip="">
-
-.. only:: html
-
- .. figure:: /auto_meta_modeling/kriging_metamodel/images/thumb/sphx_glr_plot_kriging_cantilever_beam_hmat_thumb.png
-     :alt: Create a copula
-
-     :ref:`sphx_glr_auto_meta_modeling_kriging_metamodel_plot_kriging_cantilever_beam_hmat.py`
-
-.. raw:: html
-
-    </div>
-
-.. toctree::
-   :hidden:
-
-   /auto_meta_modeling/kriging_metamodel/plot_kriging_cantilever_beam_hmat
-
```

Comment:
I have the filling that this small issue has negligible impacts (on local install only) and probably concern very few people (let us say the small community of developers). If we merge this commit, examples with `hmat` will vanish. Isn't it?
I prefer rather having local unreferenced examples. What do you think?

Reply:
its a compromise: either have a missing example or get a cryptic error about a missing ref (in case if sphinx -W + hmat not found)
I'd rather have the first one, as the hmat example is very specific anyways and probably not the first example to look

Reply:
its to avoid a sphinx error if hmat is not found by cmake

Reply:
Is it technically possible to hide the example if and only if hmat is not found?

Reply:
in theory yes with sphinx tags, but it didnt work back then for ipynb examples